x-pack/plugins/reporting/export_types/csv_from_savedobject/server/lib/index.d.ts

```diff
@@ -20,7 +20,7 @@ export interface SavedSearchGeneratorResult {
 
 export interface CsvResultFromSearch {
   type: string;
-  result: SavedSearchGeneratorResult | null;
+  result: SavedSearchGeneratorResult;
```

Comment:
It made things complicated to try to support this being `null`, since the value gets passed in `h.response` in the route handler, and `h.response` does not accept `null` per TS types.

That is why, further down, the code will check if this is null and convert to `undefined`